Sofa/Component/Mass/src/sofa/component/mass/MeshMatrixMass.cpp

```diff
@@ -48,13 +48,14 @@ Vector6 MeshMatrixMass<Vec3Types>::getMomentum ( const core::MechanicalParams*,
         for( int j=0 ; j<DataTypes::spatial_dimensions ; ++j ) momentum[3+j] += angularMomentum[j];
     }
 
+    const auto& edges = l_topology->getEdges();
     for(size_t i=0 ; i<l_topology->getNbEdges() ; ++i )
     {
-        unsigned v0 = l_topology->getEdge(i)[0];
-        unsigned v1 = l_topology->getEdge(i)[1];
+        const auto v0 = edges[i][0];
```

Comment:
remove warning and also optimization! 👍

Reply:
Yes! It would be great to catch all similar situations